src/components/ContentReveal/_content-reveal.scss

```diff
@@ -0,0 +1,43 @@
+.content-reveal {
+  & + & {
+    border-top: 1px solid #ccc;
```

Comment:
`#ccc` isn't one of our colours is it?

Reply:
See https://app.zeplin.io/project/6014403a922c684280ad7011/screen/601440698c2a070d826d41d6 or `settings.scss`